sucury.py

```diff
@@ -350,6 +353,22 @@ def grid_resize():
 
         pygame.display.update()
 
+##
+## Toggle fullscreen on game window
+##
+        
+def fullscreen():
+    global WIDTH, HEIGHT, SCREEN, width_temp, height_temp
+    if bool(SCREEN.get_flags() & pygame.FULLSCREEN): # Check if FULLSCREEN flag is activated
+        SCREEN = pygame.display.set_mode((width_temp, height_temp)) # Returns window size to original
+        WIDTH, HEIGHT = width_temp, height_temp
+        draw_grid()
+    else:
+        width_temp, height_temp = WIDTH, HEIGHT
+        SCREEN = pygame.display.set_mode((0, 0), pygame.FULLSCREEN) # Enables FULLSCREEN flag
+        WIDTH, HEIGHT = pygame.display.get_surface().get_size()
+        draw_grid()
+    
```

Comment:
There is a problem in this function. The way we are placing objects at the screen depends on the WIDTH and HEIGHT. When you die and resets at fullscreen, the game is crashing. It happens when you enter fullscreen and press start too.